crates/component-macro/src/bindgen.rs

```diff
@@ -209,7 +214,16 @@ fn parse_source(
     };
 
     let inline_pkg = if let Some(inline) = inline {
-        Some(resolve.push(UnresolvedPackage::parse("macro-input".as_ref(), inline)?)?)
+        let UnresolvedPackageGroup {
+            packages,
+            source_map,
+        } = UnresolvedPackageGroup::parse("macro-input".as_ref(), inline)?;
+        Some(
+            packages
+                .into_iter()
+                .map(|p| resolve.push(p, &source_map))
+                .collect::<anyhow::Result<Vec<_>>>()?,
+        )
```

Comment:
This can be simplified a bit by using `Resolve::append` instead of iterating over the packages and calling `push` manually, although that does the same thing too.